atlasdb-backup/src/test/java/com/palantir/atlasdb/backup/AtlasBackupServiceTest.java

```diff
@@ -116,6 +128,48 @@ public void prepareBackupThrowsIfNamespacesCollideWithExistingInProgressBackups(
                 .hasMessageContaining("Duplicated namespaces");
     }
 
+    @Test
```

Comment:
If you decide to go with my suggestion above, let's add some more tests for that.

Reply:
done